scripts/ort-evaluator.sh

```diff
@@ -11,16 +11,49 @@ else
     ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_SCANNER_FILE
 fi
 
+if [[ -s "$ORT_CONFIG_LICENSE_CONFIGURATION_FILE" ]]; then
+    ORT_EVALUATOR_OPTIONS="--license-classifications-file $ORT_CONFIG_LICENSE_CONFIGURATION_FILE"
+else
+    ORT_EVALUATOR_OPTIONS=""
+fi
+
+if [[ -d "$ORT_CONFIG_PACKAGE_CONFIGURATION_DIR" ]]; then
+    ORT_EVALUATOR_OPTIONS="$ORT_EVALUATOR_OPTIONS --package-configuration-dir $ORT_CONFIG_PACKAGE_CONFIGURATION_DIR"
+elif [[ -s "$ORT_CONFIG_PACKAGE_CONFIGURATION_FILE" ]]; then
```

Comment:
Shall we stick with `-f` instead of `-s` to keep it simpler, e.g. if the file exists it has to be in valid format? (same below)